src/LeanCode.ContractsGenerator/Program.cs

```diff
@@ -19,7 +19,7 @@ public class ProjectOptions : IOptions
     {
         public string OutputFile { get; set; }
 
-        [Option('p', "project", Required = true, Separator = ';', MetaValue = "FILE", HelpText = "The project file with contracts. To pass multiple projects, separate the paths with ';'.")]
+        [Option('p', "project", Required = true, MetaValue = "FILE", HelpText = "The project file with contracts. To pass multiple projects, separate the values with space.")]
```

Comment:
How does it work when you have paths with spaces? Is it like `--project "../path with spaces/1.csproj" "../path with spaces/2.csproj"`?

Reply:
> as split by the OS.
yes

Reply:
@konowrockis Exactly that. Each parameter as interpreted by the OS parser will be considered a separate path.